Engine/src/mini/font/ColorTags.java

```diff
@@ -104,27 +104,51 @@ void setBaseColor(ColorRGBA color) {
         Range(int start, String colorStr) {
             this.start = start;
             this.color = new ColorRGBA();
+
             if (colorStr.length() >= 6) {
-                color.set(Integer.parseInt(colorStr.subSequence(0, 2).toString(), 16) / 255f,
-                          Integer.parseInt(colorStr.subSequence(2, 4).toString(), 16) / 255f,
-                          Integer.parseInt(colorStr.subSequence(4, 6).toString(), 16) / 255f,
-                          1);
-                if (baseAlpha != -1) {
-                    color.a = baseAlpha;
-                } else if (colorStr.length() == 8) {
-                    color.a = Integer.parseInt(colorStr.subSequence(6, 8).toString(), 16) / 255f;
-                }
+                readHexColor(colorStr);
+				return;
+            }
+
+            readColor(colorStr);
+        }
+
+        private void readHexColor(String colorStr) {
+            color.r = readHexColorValue(colorStr, 0);
+            color.g = readHexColorValue(colorStr, 1);
+            color.b = readHexColorValue(colorStr, 2);
+
+            if (baseAlpha != -1) {
+                color.a = baseAlpha;
+            } else if (colorStr.length() == 8) {
+                color.a = readHexColorValue(colorStr, 3);
             } else {
-                color.set(Integer.parseInt(Character.toString(colorStr.charAt(0)), 16) / 15f,
-                          Integer.parseInt(Character.toString(colorStr.charAt(1)), 16) / 15f,
-                          Integer.parseInt(Character.toString(colorStr.charAt(2)), 16) / 15f,
-                          1);
-                if (baseAlpha != -1) {
-                    color.a = baseAlpha;
-                } else if (colorStr.length() == 4) {
-                    color.a = Integer.parseInt(Character.toString(colorStr.charAt(3)), 16) / 15f;
-                }
+                color.a = 0;
```

Comment:
That is wrong, this needs to be a 1. If you look at how color.set was used previously.